javascript-data-model/order-history.js

```diff
@@ -0,0 +1,62 @@
+const orderHistory = [
+  {
+    'order-number': '113-2883177-2648248',
```

Comment:
Property names should always be `camelCased`. You shouldn't use hypens or spaces. We want to follow best practices and conventional naming schemes. 

Instead, this should just be `orderNumber`

Reply:
Changed it, do I need to request a new pull or can I just merge?

<img width="552" alt="Screenshot 2023-08-29 at 11 35 43 AM" src="https://github.com/jeremyhnguyen/c-0823-code-solutions/assets/136545701/f7375d46-f1ba-4967-be80-93a4b0f0a1a9">